src/braket/circuits/diagram_builders/braket_circuit_drawer.py

```diff
@@ -0,0 +1,293 @@
+from typing import Any, Optional
+
+import matplotlib.pyplot as plt
+from matplotlib.patches import Circle, Rectangle
+
+PARAMETERIZED_GATES = {"RX", "RY", "RZ", "PHASE", "U3"}
+CONTROL_GATES = {"CNOT", "CX"}
+TOFFOLI_GATES = {"TOFFOLI", "CCX", "CCNOT"}
```

Comment:
I don't think we should hard-code these lists of gates - this will be limiting when new gates are added, or a user is adding custom gates. The operator objects themselves should have enough information to know how to render them. Are you running into a case where you can't figure it out? If so, let me know and I can help figure out a workaround.